### PR TITLE
Balances update spell type in expose

### PR DESCRIPTION
**Pull request by aalan3**

### CODE REVIEW

File: models/_sector/tokens/arbitrum/tokens_arbitrum_balances.sql [hunk 1]
```diff
@@ -2,10 +2,11 @@
...
         materialized = 'view',
-        post_hook = '{{ expose_spells(\'["arbitrum"]\',
-                                    "sector",
-                                    "tokens",
-                                    \'["aalan3"]\') }}'
+        post_hook = '{{ expose_spells(
+                        blockchains = \'["arbitrum"]\',
+                        spell_type = "sector",
+                        spell_name = "balances",
+                        contributors = \'["aalan3"]\') }}'
```
[the remark this answers is not in the document]
Comment (jeff-dude) [reply models/_sector/tokens/arbitrum/tokens_arbitrum_balances.sql:9]: fyi @0xRobin @Hosuke 
small enhancement to use arg names in the post hook, help users understand clearly what they are adding to each

**Comment by aalan3** [comment]
This only updates views so you can run it with `state:modified`, you might also have to for the post hook? But not 100% sure.

**Comment by jeff-dude** [comment]
> This only updates views so you can run it with `state:modified`, you might also have to for the post hook? But not 100% sure.

you're right, i'll let it go on it's own with that step

**Comment by 0xRobin** [comment]
@aalan3 if I understand this correctly, we're moving the balances models from the `tokens` sector to the `balances` sector?
I assume to make them easier to find/search in the explorer

**Comment by aalan3** [comment]
@0xRobin correct

**Pull request merged**